src/PuppeteerProcessDelegate.php

```diff
@@ -14,6 +14,12 @@ class PuppeteerProcessDelegate implements ShouldHandleProcessDelegation
      */
     public function resourceFromOriginalClassName(string $className): ?string
     {
+        if ($className === 'HTTPResponse') {
+            $className = 'Response';
+        } elseif ($className === 'HTTPRequest') {
+            $className = 'Request';
+        }
+
```

Comment:
This a smart bugfix, but the real fix should be to rename the basic PHP resources:

- [`Request`](https://github.com/rialto-php/puphpeteer/blob/f9a9c17d62076e5e5652df38d38fe26fc565b6f8/src/Resources/Request.php) => `HTTPRequest`
- [`Response`](https://github.com/rialto-php/puphpeteer/blob/f9a9c17d62076e5e5652df38d38fe26fc565b6f8/src/Resources/Response.php) => `HTTPResponse`

Reply:
Should we do this separately or as part of this?

Reply:
It should be done in the same PR since its related to (and caused by) the version upgrade. 😉

Reply:
okay, makes sense

Reply:
Yep. https://github.com/puppeteer/puppeteer/releases/tag/v3.2.0
![image](https://user-images.githubusercontent.com/34161928/97175611-d32dc680-17a4-11eb-94c1-1b8fc63256d4.png)
Caused by TypeScript issues. So they done this rename. But I like original name more...

If we rename classes, then users will have to fix their code (including us), if they use type checks :(